### ISSUE TITLE
MxNet Compilation flags

### ISSUE THREAD

**Issue by kampelmuehler** [msracver/Flow-Guided-Feature-Aggregation#8]
Hi,

for me demo.py wouldn't execute without errors, unless I compiled mxnet with the `USE_CUDA=1 USE_CUDA_PATH=/usr/local/cuda-8.0 USE_CUDNN=1` flags. Maybe you could/should add this to the README.

Cheers,
Moritz

**Comment by zhanghanduo** [comment]
Yeah, I agree with that. Also it is best to mention the code is built on 2.7.

**Comment by wangzhengshuai** [comment]
hellow~
which cudnn do you use? I use cuda-8.0 but it goes wrong as below when I am traing...thanks
[19:25:42] src/operator/convolution.cu:87: This convolution is not supported by cudnn, MXNET convolution is applied.
[19:25:42] src/operator/convolution.cu:87: This convolution is not supported by cudnn, MXNET convolution is applied.

**Comment by wangzhengshuai** [comment]
hello~
which cudnn do you use? I use cuda-8.0 but it goes wrong as below when I am traing...thanks
[19:25:42] src/operator/convolution.cu:87: This convolution is not supported by cudnn, MXNET convolution is applied.
[19:25:42] src/operator/convolution.cu:87: This convolution is not supported by cudnn, MXNET convolution is applied.
